### ISSUE TITLE
Can you share the training files?

### ISSUE THREAD

**Issue by PerdonLiu** [MadryLab/constructed-datasets#2]
First I really appreciate your work and it is really helpful to understand the essence of adversarial examples.

I met some problems when I try to re-implemented the standard testing results of the model trained with Non-Robust Cifar10 Dataset. I trained the Non-Robust Cifar10 Dataset as you said in Appendix C.2 but I only got a bad test accuracy(about 64% which is not 88%).

Could you please share your training code? Thanks in advance and I can't help saying that this paper is extremely useful!

**Comment by stefan-matcovici** [comment]
When testing the classifier with the original test set, are you using normalization?

**Comment by PerdonLiu** [comment]
Yes, data normalization(mean=[0.4914, 0.4822, 0.4465], std=[0.2023, 0.1994, 0.2010]) is used and batch normalization is turned off because of model.eval(). Now the test accuracy is only 81.5%. The resnet50 model I used is modified for cifar10: https://github.com/bearpaw/pytorch-classification/blob/master/models/cifar/resnet.py. 

I think there are some mistakes. Could you please point it out? Thanks a lot!

**Comment by yh960520** [comment]
> First I really appreciate your work and it is really helpful to understand the essence of adversarial examples.
> 
> I met some problems when I try to re-implemented the standard testing results of the model trained with Non-Robust Cifar10 Dataset. I trained the Non-Robust Cifar10 Dataset as you said in Appendix C.2 but I only got a bad test accuracy(about 64% which is not 88%).
> 
> Could you please share your training code? Thanks in advance and I can't help saying that this paper is extremely useful!

How did you get the Robust accuracy? Is it the same as the paper?

**Comment by PerdonLiu** [comment]
> > First I really appreciate your work and it is really helpful to understand the essence of adversarial examples.
> > I met some problems when I try to re-implemented the standard testing results of the model trained with Non-Robust Cifar10 Dataset. I trained the Non-Robust Cifar10 Dataset as you said in Appendix C.2 but I only got a bad test accuracy(about 64% which is not 88%).
> > Could you please share your training code? Thanks in advance and I can't help saying that this paper is extremely useful!
> 
> How did you get the Robust accuracy? Is it the same as the paper?

I only tried to reproduce the standard accuracy but what I got was not so good.
